pdir/api.py

```diff
@@ -203,7 +203,12 @@ def get_oneline_doc(self) -> str:
                 doc_list.append(doc.split('\n', 1)[0])
             return ', '.join(doc_list)
 
-        if hasattr(attr, '__doc__'):
+        try:
+            hasattr_doc = hasattr(attr, "__doc__")
```

Comment:
To be consistent, could you switch to single quote?

Reply:
done